setup.py

```diff
@@ -66,8 +52,6 @@ def cpp_flag(compiler):
                        'is needed!')
 
 
-
-
 class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
     c_opts = {
```

Comment:
Typically for pybind11 you'd use the one they have. However this seems to set very specific flags for each platform and I can't confirm if pybind11 handles this for you. 

I will leave this alone.